configure.py

```diff
@@ -1300,6 +1302,14 @@ def set_tf_cuda_compute_capabilities(environ_cp):
         if ver < 3:
           print('Only compute capabilities 3.0 or higher are supported.')
           all_valid = False
+        elif warning_flag = False and float(compute_capability) < 3.5:
+          warning_flag = True
```

Comment:
Configure script prints a lot. I fear that this may get lost among all the messages.
Should we just print the message and call sys.exit(1) here?

Reply:
The all_valid flag (which I suggested using in another comment just now) triggers a loop if it's `False`, which does something similar (although it looks like setting an improper value in an env param would make a scripted configuration time out)

Reply:
In the existing version of **configure.py**, the `all_valid` flag, essentially checks if all the compute capabilities specified are *>=3.0* (the minimum required to build TensorFlow). Now, if we are simply changing the minimum requirement to 3.5, I think that we should leave the rest of the script unchanged, and simply do a...

```
ver = float(m.group(0))
if ver < 3.5:
  print('...') # Print the appropriate error message
  all_valid = False
```
Just the way @angersson has suggested in another comment.